getting_started_julia/getting_started.ipynb

```diff
@@ -374,7 +374,7 @@
    "source": [
     "```julia\n",
     "using InstantiateFromURL\n",
-    "github_project(\"QuantEcon/quantecon-notebooks-julia\", version = \"0.8.0\", instantiate = true, precompile = true)\n",
+    "github_project(\"QuantEcon/quantecon-notebooks-julia\", version = \"0.8.0\", force = true)\n",
```

Comment:
There was a change in instantiatefromurl to add a new feature for precompilation control in https://github.com/QuantEcon/InstantiateFromURL.jl/releases/tag/v0.5.0  

So my guess is that you have an older version of that package and that a `] up InstantiateFromURL` (done OUTISIDE of the activated quantecon notebooks environment) could do the trick.  Then you could delete the notebooks, follow the setup instructions, and I bet it works great.

The issue with `force` is that it forces an overwrite and download - which is generally not what you want when you have a cloned notebook repository.

Feel free to post on discourse.quantecon.org to discuss.

Also note that our source to generate these notebooks is in https://github.com/QuantEcon/lecture-source-jl  so if you have found a bug we fix it there.

Reply:
@jlperla fresh julia install and getting this error. `up InstantiateFromURL` didn't help either. I understand why you split `force` into two separate parameters, but it sounds like package publishing didn't work out?

Feel free to close this pull request and address it in the source repo. I was actually wondering how you maintain a collection of ipython notebooks - now I know :)

Reply:
Thanks. @arnav sorry to bug you, but any ideas with your InstantiateFromURL updates? Did you have issues when you did the local testing with the v0. 8?